laborator/content/linking/README.md

```diff
@@ -406,3 +345,23 @@ quantity is 42
 ```
 
 Explorați interfața și conținutul funcțiilor din fisierul `shop.o` folosind `nm` și `objdump`.
+
+### Bonus. Utilizare cod python în C
```

Comment:
In masina virtuala nu merge comanda `make`:
Python.h: No such file or directory